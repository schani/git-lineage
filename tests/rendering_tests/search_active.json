{
  "active_panel": "Navigator",
  "file_tree": [
    {
      "name": "config.toml",
      "path": "config.toml",
      "is_dir": false,
      "git_status": "M",
      "children": []
    },
    {
      "name": "src",
      "path": "src",
      "is_dir": true,
      "git_status": null,
      "children": [
        {
          "name": "config.rs",
          "path": "src/config.rs",
          "is_dir": false,
          "git_status": "A",
          "children": []
        }
      ]
    }
  ],
  "selected_file_path": null,
  "search_query": "config",
  "in_search_mode": true,
  "commit_list": [],
  "selected_commit_index": null,
  "current_content": [],
  "cursor_line": 0,
  "cursor_column": 0,
  "inspector_scroll_vertical": 0,
  "inspector_scroll_horizontal": 0,
  "show_diff_view": false,
  "status_message": "Search: config",
  "is_loading": false
}