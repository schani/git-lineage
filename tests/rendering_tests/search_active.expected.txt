┌ File Navigator (Search: c┐┌ Code Inspector ──────────────────────────────────┐
│M config.toml             ││Select a file to view its content                 │
│▶ src                     ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
└──────────────────────────┘│                                                  │
┌ Commit History ──────────┐│                                                  │
│Select a file to view its ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
│                          ││                                                  │
Search: config | Tab: Switch panel | 1/2/3: Direct panel focus | /: Search | ↑↓:
